README.md

```diff
@@ -125,50 +153,56 @@ The following example will show you how to:
 
 ### Changing a drawing handlers options
 
-You can change a draw handlers options after initialisation by using the `setDrawingOptions` method on the Leaflet.draw control.
+You can change a draw handlers options after initialization by using the `setDrawingOptions` method on the Leaflet.draw control.
 
-E.g. to change the colour of the rectangle:
+E.g. to change the color of the rectangle:
 
 ```js
 drawControl.setDrawingOptions({
     rectangle: {
-    	shapeOptions: {
-        	color: '#0000FF'
+        shapeOptions: {
+            color: '#0000FF'
         }
     }
 });
 ```
 
-# Contributing
- 
-## Testing
+## Contributing
+
+### Testing
 
 To test you can install the npm dependencies:
 
-    npm install
+```sh
+npm install
+```
 
 and then use:
 
-    jake test
+```sh
+jake test
```

Comment:
This should be `yarn test`

Reply:
Ditto for just applying Markdown Validator fixes, not changing content of the original.